src/contacts/processors/impl/SearchAgainRecordProcessor.java

```diff
@@ -0,0 +1,47 @@
+package contacts.processors.impl;
+
+import contacts.ConsoleReader;
+import contacts.model.Record;
+import contacts.processors.IActionProcessor;
+import contacts.service.impl.IRecordService;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class SearchAgainRecordProcessor implements IActionProcessor {
+
+    private final IRecordService recordService;
+
+    public SearchAgainRecordProcessor(IRecordService recordService) {
+        this.recordService = recordService;
+    }
+
+    @Override
+    public boolean doAction() throws IOException, ClassNotFoundException {
```

Comment:
how can it throw ClassNotFoundException?

Reply:
Done, deleted

Reply:
When I use classes for writing or reading from File, it can throws ClassNotFoundException